src/Grids/input_validation.jl

```diff
@@ -123,7 +124,7 @@ function validate_dimension_specification(T, ξ::AbstractVector, dir, N, FT)
 end
 
 function validate_dimension_specification(T, ξ::Function, dir, N, FT)
```

Comment:
Is this method needed in the end? - I thought the idea of #3331 was to eliminate validating until after the coords is generated.

Reply:
No, it's not needed